tests/utils/MinimumSpanningTreeTest.cpp

```diff
@@ -1,144 +1,135 @@
-//Copyright (c) 2019 Ultimaker B.V.
-//CuraEngine is released under the terms of the AGPLv3 or higher.
-
-#include <gtest/gtest.h>
+//  Copyright (c)  2019-2022 Ultimaker B.V.
+//  CuraEngine is released under the terms of the AGPLv3 or higher.
 
+#include "utils/MinimumSpanningTree.h"
 #include <algorithm>
-#include <vector>
+#include <gtest/gtest.h>
 #include <unordered_map>
-
-#include "../src/utils/MinimumSpanningTree.h"
+#include <vector>
 
 namespace cura
 {
-    bool has(const Point& pt, const std::vector<Point>& list)
-    {
-        return std::find(list.begin(), list.end(), pt) != list.end();
-    }
+bool has(const Point& pt, const std::vector<Point>& list)
+{
+    return std::find(list.begin(), list.end(), pt) != list.end();
+}
 
-    class MinimumSpanningTreeTest : public ::testing::Test
-    {
-    public:
-        void SetUp() override
-        {
-            pts =
-            {
-                Point(-3, -4),
-                Point( 3, -4),
-                Point( 0, -3),
-                Point( 0,  0),
-                Point( 1,  1),
-                Point( 5,  1),
-                Point(-1,  6),
-                Point( 0,  5),
-                Point( 5,  7),
-                Point(12, 12),
-                Point(12, 13),
-            };
-            std::vector<Point> pts_set(pts.begin(), pts.end());
-            p_mst = new MinimumSpanningTree(pts_set);
-        }
-
-        void TearDown() override
-        {
-            delete p_mst;
-            p_mst = nullptr;
-        }
-
-        std::vector<Point> pts;
-        MinimumSpanningTree* p_mst; // Needs to be a pointer, beacuse SetUp isn't a constructor and the copy function is deleted.
-    };
-
-    TEST(SimpleMinimumSpanningTreeTest, TestConstructEmpty)
+class MinimumSpanningTreeTest : public ::testing::Test
+{
+  public:
+    void SetUp() override
     {
-        std::vector<Point> vertices;
-        MinimumSpanningTree tree(vertices);
-
-        ASSERT_TRUE(tree.leaves().empty()) << "Empty tree should be empty.";
+        pts = {
+            Point(-3, -4), Point(3, -4), Point(0, -3), Point(0, 0),   Point(1, 1),   Point(5, 1),
+            Point(-1, 6),  Point(0, 5),  Point(5, 7),  Point(12, 12), Point(12, 13),
```

Comment:
Why are these two lines aligned like this? They don't belong vertically to each other.

The X and Y coordinates used to be vertically aligned, but why did you remove that?